README.md

```diff
@@ -25,6 +27,10 @@ config :pool_max, :validate => :number, :default => 50
 # Max number of concurrent connections to a single host. Defaults to 25
 config :pool_max_per_route, :validate => :number, :default => 25
 
+# Specify this if you want to keep the TCP connection open between requests
+# using HTTP Keep-Alive. Defaults to 0 (off)
+config :keepalive, :validate => :number, :default => 0
```

Comment:
I didn't do an exhaustive search, but in manticore it seems like keepalive is treated as a boolean:

https://github.com/cheald/manticore/blob/ca857e63c717aae4f91bf66f61643dd14a988ee5/lib/manticore/client.rb#L178